packages/docs-site/.gitignore

```diff
@@ -18,3 +18,7 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# Include style docs folder
+!Style
+!Substance
```

Comment:
Could you clarify what exactly these are referring to? I don't see any files or folders in this PR whose name is `Style` or `Substance` (with a leading capital S).